abandonauth/routers/password_login.py

```diff
@@ -0,0 +1,51 @@
+from abandonauth.settings import settings
+from abandonauth.dependencies.auth.jwt import generate_long_lived_jwt
+from fastapi import APIRouter, Response
+
+from abandonauth.dependencies.auth.hash import (
+    get_hashed_data, verify_data
+)
+from prisma.models import PasswordAccount, User
+from abandonauth.models.user import PasswordAccountSchema, PasswordLoginDto
+
+router = APIRouter()
+
+
+@router.post('/create_test_user', summary='Creates User and PasswordAccount')
+async def create_test_user(user_data: PasswordAccountSchema):
+    user = await User.prisma().find_first(
+        where={
+            "password_account": {
+                "is": {
+                    "password": get_hashed_data(user_data.password)
+                }
+            }
+        }
+    )
+
+    if user is None:
+        user = await User.prisma().create({
+            "username": user_data.username,
+            "password_account": {
+                "create": {
+                    "password": get_hashed_data(user_data.password)
+                }
+            }
+        })
+    return {'user_id': user.id}
+
+
+@router.post('/login_test_user', summary='Login using password')
+async def login_test_user(user_data: PasswordLoginDto, res: Response):
+    password_account = await PasswordAccount.prisma().find_first(
+        where={
+            "user_id": str(user_data.user_id)
+        }
+    )
+    if password_account is None:
+        return {'message': 'User not found'}
+    if not verify_data(user_data.password, password_account.password):
+        return {'message': 'Invalid password'}
```

Comment:
We should raise an exception here. We can also duplicate the more secure logic we already have for developer applications. (Though the user and password checks are combined to prevent brute-forcing/data-mining and are admittedly not really necessary on this dev endpoint)

```suggestion
  if password_account is None or not verify_data(user_data.password, password_account.password):
        raise HTTPException(
            status_code=HTTP_401_UNAUTHORIZED,
            detail="Invalid username or password",
        )
```